### ISSUE TITLE
grammar confused by multiple structs

### ISSUE THREAD

**Issue by djacobow** [mohawk2/Inline-Struct#3]
It seems that I can't have more than one struct in my code because the grammar gets confused and runs the names together.

Error:

> Could not find a typemap for C type 'struct bbb_t *'.
> The following C types are mapped by the current typemap:
> 'AV *', 'Boolean', 'CV *', 'FILE *', 'FileHandle', 'HV *', 'I16', 'I32', 'I8', 'IV', 'InOutStream', 'InputStream', 'NV', 'OutputStream', 'PerlIO *', 'Result', 'STRLEN', 'SV *', 'SVREF', 'SysRet', 'SysRetLong', 'Time_t *', 'U16', 'U32', 'U8', 'UV', 'aaa_t *', 'bbb_t *', 'bool', 'bool_t', 'caddr_t', 'char', 'char *', 'char **', 'const char *', 'double', 'float', 'int', 'long', 'short', 'size_t', 'ssize_t', 'struct aaa_t * O_OBJECT_aaa_tstruct bbb_t *', 'time_t', 'unsigned', 'unsigned char', 'unsigned char *', 'unsigned int', 'unsigned long', 'unsigned long *', 'unsigned short', 'void *', 'wchar_t', 'wchar_t *'
>  in failex_eec2.xs, line 165

Note "struct aaa_t *O_OBJECT_aaa_tstruct bbb_t *"

aaa_t and bbb_t are separate things, but in the typemap you create they get smushed.

Here is the complete example:

```
#!/usr/bin/perl -w

use strict;
use warnings qw(all);

package failex;

use Inline C => Config => enable => "autowrap";
use Inline C => 'DATA', structs => ['aaa_t','bbb_t'];

my $aaa = Inline::Struct::aaa_t->new;
my $bbb = Inline::Struct::bbb_t->new;

foo($aaa,$bbb);

__DATA__
__C__

struct aaa_t {
   int a;
   double b;
};

typedef struct aaa_t aaa_t;

struct bbb_t {
    float b;
    float c;
};

typedef struct bbb_t bbb_t;


void foo(aaa_t *v1, bbb_t *v2) {
    v2->b = v1->b;
};

```

**Comment by mohawk2** [comment]
Thanks, fixed!

**Comment by mohawk2** [comment]
...and released as 0.24. Let me know any problems.
